examples/run/taskrun-docker-basic.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: test-docker-basic
+  annotations:
+    build.knative.dev/docker-0: https://us.gcr.io
+    build.knative.dev/docker-1: https://eu.gcr.io
+    build.knative.dev/docker-2: https://asia.gcr.io
+    build.knative.dev/docker-3: https://gcr.io
+    build.knative.dev/docker-4: https://reduce-chance-of-selecting-gcr.io
```

Comment:
i think these should be something else, maybe [`pipeline.knative.dev`](https://github.com/knative/build-pipeline/blob/master/docs/auth.md#guiding-credential-selection) ? (not sure if this actually works tho 😅 )

Reply:
It does not work. Is this something we need a separate issue or tackle it in this PR?

Reply:
oh yeah separate PR then for sure i think! separate issue even